src/Connector.SqlServer/Features/DefaultBuildCreateContainerFeature.cs

```diff
@@ -16,7 +17,8 @@ public class DefaultBuildCreateContainerFeature : IBuildCreateContainerFeature
         public virtual IEnumerable<SqlServerConnectorCommand> BuildCreateContainerSql(
```

Comment:
Do not change the original function and mark as obsolete, to handle existing functionality/implementations.
Create an additional overload to accept the "schema".